modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -124,15 +124,16 @@ object GlobalSnapshotConsensusFunctions {
           maybeTipTrigger.fold(lastGS.height)(_.height),
           maybeTipTrigger.fold(lastGS.subHeight.next)(_ => SubHeight.MinValue),
           lastGSHash,
-          blocksInRange,
+          blocksInRange.map(BlockAsActiveTip(_, 0L)),
           scSnapshots,
           rewards,
           NonEmptyList.of(PeerId(Hex("peer1"))), // TODO
           GlobalSnapshotInfo(
             lastGS.info.lastStateChannelSnapshotHashes ++ sCSnapshotHashes,
             (lastGS.info.lastTxRefs ++ lastTxRefs).groupMapReduce(_._1)(_._2) { case (_, updated) => updated },
             (lastGS.info.balances ++ balances).groupMapReduce(_._1)(_._2) { case (_, updated)     => updated }
-          )
+          ),
+          lastGS.tips
```

Comment:
I guess the logic filtering moving the tips to deprecated will come later, right?

Reply:
Yep, in the next PR.